samples/xmpsample.cpp

```diff
@@ -10,6 +10,8 @@
 #include <cassert>
 #include <cmath>
 
+#define UNUSED(expr) do { (void)(expr); } while (0)
```

Comment:
I think this is then the 4th `UNUSED` macro in the code base ;-)

Reply:
:fearful: ! I did not notice. I will move this macro to a implementation header and remove the other ones.